README.md

```diff
@@ -1,18 +1,99 @@
 ## Chess
 
-Building a platform where people can
+Welcome to the Chess Repository! This platform allows users to sign up, create new matches, get connected to existing matches, and play real-time chess games.
 
-1. Sign up
-2. Create a new match/get connected to an existing match
-3. During the match, let users play moves
-4. Have a rating system that goes up and down similar to standard chess rating
+## Installation
 
-## Tech stack
+To get started, follow these steps:
 
-Let's keep it simple
+1. Clone this repository to your local machine:
+```bash
+git clone https://github.com/code100x/chess.git
+```
 
-1. React for Frontend
-2. Node.js for Backend
-3. Typescript as the language
-4. Saparate Websocket servers for handling real time games
-5. Redis for storing all moves of a game in a queue
+2. Navigate to the project directory:
+```bash
+cd chess
+```
+
+3. Install dependencies for both frontend and backend:
+```bash
+cd apps/frontend/
+npm install
+
+cd apps/backend/
+npm install
+
+cd apps/ws/
+npm install
+
+cd packages/db/
+npm install
```

Comment:
you don't need go to different directories and do npm install. Doing it on the root directory would work

Reply:
+1

Reply:
> you don't need go to different directories and do npm install. Doing it on the root directory would work

could you check now?